algorithms/kernel/kernel.h

```diff
@@ -83,7 +92,7 @@
                             DAAL_KERNEL_AVX512_CONTAINER(ContainerTemplate, __VA_ARGS__)>::ClassName(daal::services::Environment::env * daalEnv)    \
         : BaseClassName(daalEnv), _cntr(nullptr)                                                                                                    \
     {                                                                                                                                               \
-        switch (daalEnv->cpuid)                                                                                                                     \
+        GetCpuid switch (cpuid)                                                                                                                     \
```

Comment:
should be here separate line for `cpuid` initialization?

Reply:
That is result of applying clang-format to this code. If we separate this line it will probably break our clang-format rules.